v2/spanner-to-sourcedb/terraform/samples/spanner-to-sharded-mysql/README.md

```diff
@@ -0,0 +1,508 @@
+## Sample Scenario: Spanner to Sharded MySQL reverse replication
+
+> **_SCENARIO:_** This Terraform example illustrates launching a reverse replication
+> jobs to replicate spanner writes for a sharded MySQL source, setting up all the required cloud infrastructure.
+> **Only the source details of physical shards are needed as input.**
```

Comment:
Is this correct? We will need logical shards info as well?

Reply:
Makes sense; rephrased to `MySQL shards` since that would make the most sense for a sharded MySQL customer.